src/features/modals/CreateToken/CreateTokenBody/CreateTokenBody.tsx

```diff
@@ -0,0 +1,71 @@
+//- React Imports
+import React, { FC } from 'react';
+
+//- Component Imports
+import { DetailsForm } from './DetailsForm/DetailsForm';
+import { TokenomicsForm } from './TokenomicsForm/TokenomicsForm';
+import { TokenSummary } from './TokenSummary/TokenSummary';
+
+//- Type Imports
+import { DetailsFormSubmit, TokenomicsFormSubmit } from '../CreateToken.types';
+import { MediaType } from '@zero-tech/zui/src/components/MediaInput';
+
+type CreateTokenBodyProps = {
+	stepId: string;
+	detailsFormValues: DetailsFormSubmit;
+	onDetailsSubmit: (values: DetailsFormSubmit) => void;
+	tokenomicsFormValues: TokenomicsFormSubmit;
+	onTokenomicsSubmit: (values: TokenomicsFormSubmit) => void;
+	onMediaInputChange: (
+		mediaType: MediaType,
+		previewUrl: string,
+		image: Buffer,
+	) => void;
+	onLaunchSubmit: () => void;
+	onClose: () => void;
```

Comment:
Could be beneficial to use ContextAPI here so you don't have to pass as many props around. Not required yet, I don't think, but I think it will make future iteration easier.

Reply:
Yep, agree. Great call!